src/request/components/request-view.scss

```diff
@@ -377,54 +377,96 @@
 }
 
 .tab-logs {
-    .fa-svg {
-        height: 12px;
-        width: 12px;
-        margin-right: 5px;
-        vertical-align: middle;
+    .tab-logs-message-count {
+        font-family: 'Segoe UI', 'Selawik Semibold', Tahoma, Geneva, Verdana, sans-serif;
+        font-size: 175%;
     }
-}
 
-.tab-logs-data-replaced-region {
-    font-weight: bold;
-}
+    .tab-logs-table {
+        table-layout: fixed;
+        
+        .fa-svg {
+            height: 12px;
+            width: 12px;
+            margin-right: 3px;
+            vertical-align: middle;
+        }
 
-.tab-logs-message-count {
-    font-family: 'Segoe UI', 'Selawik Semibold', Tahoma, Geneva, Verdana, sans-serif;
-    font-size: 175%;
-}
+        .tab-logs-table-message {
+            display: flex;
+            flex-direction: row;
+            
+            .tab-logs-table-message-icon {
+                visibility: hidden;               
+            }
 
-.tab-logs-data-default {
-    font-weight: normal;
-    color: #333333; // grey
-    vertical-align: middle;
-}
+            .tab-logs-table-message-icon-expandable {
+                @extend .tab-logs-table-message-icon;
+                
+                visibility: visible;               
+            }
 
-.tab-logs-data-warning {
-    @extend .tab-logs-data-default;
-    
-    color: #D35400; // orange
-    
-    .fa-svg {
-        .fa-svg-path {        
-          fill: #D35400; // orange
+            &:hover {
+                .tab-logs-table-message-icon {
+                    visibility: visible;               
+                }
+            }
+ 
+            .tab-logs-table-message-object {
+                overflow: auto;
+                
+                > pre {
+                        margin: 0;
+                        
+                        > code {
+                            background: transparent;
+                            padding: 0;
+                        }
+                    }
+            }
```

Comment:
Thanks, I'll take a look.  I agree that we'll need to do some CSS refactoring as we redesign some of the major Client components and intend to share more common elements.  One of the things that I've wondered about are how to minimize the coupling within the code and the CSS while still allowing for a layered approach to styling.  (For example, I noticed that the log table adds a `table-striped` value to the class name and it seemed a little odd to me to have the code referring to what should be a purely stylistic concept.)